frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -156,6 +157,15 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
             LoginRequest request,
             UserContext userContext) {
 
+        var optionalAuthSession =
+                authSessionService.getSessionFromRequestHeaders(input.getHeaders());
+
+        if (optionalAuthSession.isEmpty()) {
```

Comment:
Similar comment to with the other handler - should this be 400 when Session-Id is missing, 500 otherwise?